plugins/UsersManager/API.php

```diff
@@ -922,7 +922,9 @@ public function updateUser($userLogin, $password = false, $email = false, $alias
             $this->sendEmailChangedEmail($userInfo, $email);
         }
 
-        if ($passwordHasBeenUpdated) {
+        if ($passwordHasBeenUpdated
+            && $requirePasswordConfirmation
```

Comment:
not tested but looks good 👍

Reply:
Patch tested and no e-mail is sent after login with a ldap account.

Reply:
@jgkirschbaum  thx for confirmation